test/unit/operation_test.js

```diff
@@ -39,6 +39,17 @@ describe('Operation', function() {
       );
     });
 
+    it('creates a createAccount operation with startingBalance equal to 0', function() {
+      let opts = {
+        destination: 'GCEZWKCA5VLDNRLN3RPRJMRZOX3Z6G5CHCGSNFHEYVXM3XOJMDS674JZ',
+        startingBalance: '0',
+        source: 'GCEZWKCA5VLDNRLN3RPRJMRZOX3Z6G5CHCGSNFHEYVXM3XOJMDS674JZ'
+      };
+      expect(() => StellarBase.Operation.createAccount(opts)).not.to.throw(
+        /startingBalance argument must be of type String, represent a positive number and have at most 7 digits after the decimal/
```

Comment:
Shouldn't we change this error msg too (include zero).

Reply:
We can say "non-negative" instead of "positive" to include `>= 0`.

Reply:
good point.

Reply:
message updated https://github.com/stellar/js-stellar-base/commit/4efa3a853c25201643deb3b75b4cc830e5b8aa2e